Procfile

```diff
@@ -1,2 +1,2 @@
-web:       gunicorn cabot.wsgi:application --config gunicorn.conf
-celery:    celery worker -B -A cabot --loglevel=INFO --concurrency=16 -Ofair
+web:       gunicorn cabot.wsgi:application --config gunicorn.conf --log-file -
+celery:    celery worker -B -A cabot --loglevel=INFO --concurrency=16 -Ofair 2>&1
```

Comment:
I think flynn (and Heroku?) only log stdout, not stderr. I therefore redirect stderr to stdout.

`--log-file -` sends gunicorn output to stdout, which can be captured by heroku/flynn for logging.